libdnf/sack/query.cpp

```diff
@@ -63,20 +63,42 @@ struct default_delete<DnfPackage> {
 namespace libdnf {
 
 struct NevraID {
-    Id name;
```

Comment:
Code can be inline. We need initialize Ids to "0" only in default (non parameterized) constructor.
```
    NevraID() : name(0), arch(0), evr(0) {};
    NevraID(const NevraID & src) = default;
    NevraID(NevraID && src) noexcept = default;
    NevraID & operator=(const NevraID & src) = default;
    NevraID & operator=(NevraID && src) = default;
    Id name;
    Id arch;
    Id evr;
    std::string evr_str;
```